docker-compose.yml

```diff
@@ -101,7 +104,7 @@ services:
                 # CAREFUL: This must be accessible from inside the keycloak container
                 # for server-to-server communication.
                 # Use docker.for.mac.localhost as RENGA_ENDPOINT for example.
-                'site' => '${RENGA_ENDPOINT}/auth/',
+                'site' => 'http://keycloak.${PLATFORM_DOMAIN}:8080/auth',
```

Comment:
`http://keycloak.${PLATFORM_DOMAIN}/auth`

Reply:
no keycloak exposes 8080 -- if you can find an option to change the port it listens on, I'm all for changing it to 80 :)

Reply:
I suggest to make an issue about the keycloak port and move on. 8080 is not ideal, but this is a detail in my view